src/routers/GetUsersRouter.ts

```diff
@@ -14,7 +14,7 @@ class GetUsersRouter extends AuthenticatedAppplicationRouter<SerializedUser[]> {
 
   async content(req: Request): Promise<SerializedUser[]> {
     const users = await UserRepo.getUsers();
-    return users.map(userObject => userObject.serialize());
+    return users.map((userObject) => userObject.serialize());
```

Comment:
I'm assuming the linter wasn't converting interestObject variables into interest as above so you probably missed converting this one to just user